StarCraft2.ahk

```diff
@@ -25,6 +25,8 @@
 
 ; CapsLock & Numpad0: toggle Templar
 
+; CapsLock & Numpad6: toggle Cyro
```

Comment:
This comment should go a few rows lower: just below `; CapsLock & Numpad7: toggle Medic`, because i have ordered them by `Numpadx` decreasingly.